images/win/scripts/Installers/Install-Newman.ps1

```diff
@@ -0,0 +1,6 @@
+################################################################################
+##  File:  Install-Newman.ps1
+##  Desc:  Install Latest Newman
+################################################################################
+
+npm install -g newman
```

Comment:
May be it makes sense to include this installation to existing script instead of creation the new one?
https://github.com/actions/virtual-environments/blob/master/images/win/scripts/Installers/Install-NodeLts.ps1

Reply:
I'm fully agree, reworked.